lib/test/s-tooltip.test.ts

```diff
@@ -0,0 +1,62 @@
+import { html, fixture, expect } from "@open-wc/testing";
+import { screen, waitForElementToBeRemoved } from "@testing-library/dom";
+import userEvent from "@testing-library/user-event";
+import "../ts/index";
+
+const user = userEvent.setup();
+
+describe("s-tooltip", () => {
+    it("should make the tooltip element visible on hover (after a delay)", async () => {
+        const trigger = await fixture(html`
+            <button
+                class="s-btn s-btn__filled"
+                role="button"
+                data-controller="s-tooltip"
+                title="tooltip content"
+                data-s-tooltip-placement="bottom-start"
+            >
+                Hover tooltip popover
+            </button>
+        `);
+
+        expect(screen.queryByRole("tooltip")).to.be.null;
```

Comment:
Ooooh, I like this! Forcing the tests to use the accessibility tree is also a way to force a11y testing in general.